test/WebJobs.Script.Tests.Integration/packages.config

```diff
@@ -88,6 +89,7 @@
   <package id="System.Dynamic.Runtime" version="4.3.0" targetFramework="net46" />
   <package id="System.Globalization" version="4.3.0" targetFramework="net46" />
   <package id="System.Globalization.Calendars" version="4.3.0" targetFramework="net46" />
+  <package id="System.IdentityModel.Tokens.Jwt" version="5.1.3" targetFramework="net46" />

```

Comment:
This wasn't building properly for me so I re-installed the packages and this one popped in. We may have missed it in an earlier update.